config/plugins/interactive_environments/phinch/templates/phinch.mako

```diff
@@ -0,0 +1,49 @@
+<%namespace name="ie" file="ie.mako" />
+<%
+
+import os
+
+# Sets ID and sets up a lot of other variables
+ie_request.load_deploy_config()
+
+# Define a volume that will be mounted into the container.
+# This is a useful way to provide access to large files in the container,
+# if the user knows ahead of time that they will need it.
+user_file = ie_request.volume(hda.file_name, '/home/Phinch/data/'+os.path.basename(hda.file_name)+'.biom', how='ro')
```

Comment:
I actually don't have all that much experience with these datasets myself, but the example dataset contained 95 samples and is about 15 MB, so I'm guessing they never get too enormous, so bioblend option sounds good, will have a look at the way Jupyter does it.